src/lib/rmt/wizard_rmt_service_page.rb

```diff
@@ -69,17 +69,22 @@ def back_handler
 
   def run
     render_content
-    ok = false
+    rmt_services_started = false
     Yast::Popup.Feedback(_('Starting services'), _('Starting RMT server, sync, and mirror timers...')) do
-      ok = rmt_service_start
+      rmt_services_started = rmt_service_start
+      nginx_service_reload
```

Comment:
_It works when it works, but when it doesn't, it doesn't tell me_ :wink: 

I see a little bit of a mix of concerns here. So while it does what it is supposed to (reloading that service), it does it in a code block that talks about "starting timers". And this block has error handling, but that doesn't cover the service reload that you "smuggled in" here.
So to make this code more clean, I would try a version where the `nginx_service_reload` method is wrapped in it's own _open popup / do it / report error if needed_ block. That might be some duplication first, but I think it is worth it, since it tells the user what is going on, or going wrong. And probably, afterwards you could even refactor the duplication in a reusable thing. Not necessarily in this PR. I would do it as soon as we need it in a third place :smiley: